### ISSUE TITLE
Rule assignment to permission fails

### ISSUE THREAD

**Issue by Mcmester92** [johnitvn/yii2-rbac-plus#9]
Hello!

We started to use your RBAC plus extension, and at the beginning it worked fine, but after the updates the assignment fails. We tracked the error message, and it seems that the beforeSave() and afterSave() methods cause the crash, in yii2-rbac-plus/src/models/AuthItem.php line 98 and 123. I attached 2 screenshots, about the issue.

Thanks in advance!

![screenshot_26](https://cloud.githubusercontent.com/assets/12207743/9715658/b395b6b6-5562-11e5-93db-22f5247c5666.png)
![screenshot_27](https://cloud.githubusercontent.com/assets/12207743/9715659/b397dfcc-5562-11e5-8315-f4be93b54845.png)

**Comment by johnitvn** [comment]
I have update with [this](https://github.com/johnitvn/yii2-rbac-plus/commit/dd4ceba2ab96622076048f7f4011cfe2da61aa82)

Can you try to edit file `vendor/johnitvn/yii2-rbac-plus/src/models/AuthItem.php` add comment to line 98.

Tell me if it worked

**Comment by Mcmester92** [comment]
Yes it is working. Thanks for your fast help, and we are looking forward for your hotfix :)

**Comment by johnitvn** [comment]
I have release v1.0.4

**Comment by Mcmester92** [comment]
Thanks! :)
2015.09.07. 14:58 ezt írta ("John Martin" <notifications@github.com>):

**Comment by Mcmester92** [comment]
Hmm now that i checked it, I see that you only disabled beforeSave() method. I wrote in the issue, that afterSave() method kills the code too :S I commented out from the code, as you can see on the attachment, and now it finally works. When you asked me yesterday to try it, afterSave() was already commented out, and I forgot to mention that. It is my bad, sorry for that. 
![screenshot_28](https://cloud.githubusercontent.com/assets/12207743/9728342/749a85f4-5607-11e5-8430-050376385375.png)

**Comment by johnitvn** [comment]
I have been update. I will take time for upgrade this extension soon, but not at the moment. 
I'm going to make it more stable

**Comment by Mcmester92** [comment]
Thanks again! :)

2015-09-08 17:52 GMT+02:00 John Martin <notifications@github.com>:

**Comment by junaidfayyaz945** [comment]
@johnitvn  I am facing following problem. pls help me getting out of this.
![capture](https://user-images.githubusercontent.com/20260972/29504812-ddf4e0f4-865b-11e7-96e4-4a747bf70cad.PNG)

**Comment by tihoho** [comment]
> I have been update. I will take time for upgrade this extension soon, but not at the moment.
> I'm going to make it more stable

wheeeeeen? ^)
